### ISSUE TITLE
Discussion: button-name changes

### ISSUE THREAD

**Issue by jkodu** [auto-wcag/auto-wcag#264]
`aria-hidden` button element, should be `inapplicable`.

**Comment by WilcoFiers** [comment]
Taking it off the agenda. We have 19 items, need to cut some stuff off. This needs a proposal first.

**Comment by annethyme** [comment]
To have buttons that are `aria-hidden` be inapplicable, we would have to change the Applicability to not include elements that are `visible`.
Is this something we want to do?

**Comment by annethyme** [comment]
Same applies to links: https://auto-wcag.github.io/auto-wcag/rules/SC2-4-4-link-has-name.html
